### ISSUE TITLE
Way to delete plants, subplots or tissue ids?

### ISSUE THREAD

**Issue by ch728** [solgenomics/sgn#5020]
Is there any way to delete these stocks outside of https://github.com/solgenomics/sgn/blob/master/bin/delete_stocks.pl ? I assume if I delete plants any phenotypic data or tissue ids associated with the plants or associated tissue would also be wiped?

**Comment by lukasmueller** [comment]
Yes, that's why deletion is restricted on the web interface. You really want to be sure that you don't remove any other data...

**Comment by lukasmueller** [comment]
These items can be obsoleted, but not deleted.
